demos/common/cpp/utils/include/utils/ocv_common.hpp

```diff
@@ -183,6 +182,17 @@ static UNUSED ov::Tensor wrapMat2Tensor(const cv::Mat& mat) {
     return ov::Tensor(ov::element::u8, ov::Shape{ 1, height, width, channels }, mat.data);
 }
 
+static inline void resize2tensor(const cv::Mat& mat, const ov::Tensor& tensor) {
+    static const ov::Layout layout{"NHWC"};
+    const ov::Shape& shape = tensor.get_shape();
+    cv::Size size{int(shape[ov::layout::width_idx(layout)]), int(shape[ov::layout::height_idx(layout)])};
+    assert(tensor.get_element_type() == ov::element::u8);
```

Comment:
why you use assert for validating parameters ? is not it better to have just regular check for params and explicitly throw exception?

Reply:
Normally these check's aren't needed: `PrePostProcessor` should do its job. So they are removed in release build but kept in debug

Reply:
ok